ejcIMC.py

```diff
@@ -33,10 +51,11 @@ def whileErrorNumbers(validation):
         value = numbersValidation(heigth)
         value2 = numbersValidation(weigth)
```

Comment:
I suggest change this name to weightValidation or something like that... value2 say nothing to reffer about that variable